retroshare-gui/src/gui/Identity/IdDialog.ui

```diff
@@ -895,9 +891,12 @@ border-image: url(:/images/closepressed.png)
                </item>
                <item row="11" column="1">
                 <widget class="QCheckBox" name="autoBanIdentities_CB">
-                 <property name="text">
+                 <property name="toolTip">

```

Comment:
The new title "Auto Ban" is a bit misleading since we don't knw how it works (except when reading the tooltip). I understand that the previous title was too long. Maybe we find find a compromise (more informative, but short)? Maybe "Auto ban profile"? It's not so good, and I have no real suggestion.